packages/fluent-theme/src/fluent/styles/Dialog.styles.ts

```diff
@@ -1,21 +1,9 @@
 import { IDialogContentStyleProps } from 'office-ui-fabric-react/lib/Dialog';
 import { FontWeights } from 'office-ui-fabric-react/lib/Styling';
 
-import { Depths } from '../FluentDepths';
 import { FontSizes } from '../FluentType';
 import { fluentBorderRadius } from './styleConstants';
 
-export const DialogStyles = {
-  main: {
-    selectors: {
-      '.ms-Modal.ms-Dialog &': {
```

Comment:
@mikewheaton I removed this styles on `Dialog` and added them directly on `Modal` to remove the necessity to target global classNames. Currently, `Dialog` is the only component using `Modal` so I believe this should not affect anyone else. Plus I confirmed with @betrue-final-final that `Modal` should have Depth64 and rounded corners.

Reply:
Thanks, this is a far better way of doing it!